charts/infra-apps/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: infra-apps
 description: Argo CD app-of-apps config for infrastructure components
 type: application
-version: 0.77.1
+version: 0.77.2
```

Our standard policy is to bump the minor version of our chart of we bump any version of a chart included in the update:
```suggestion
version: 0.78.0
```

---

pushed changes